### PR TITLE
First draft of IPFS Weekly #64

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft by Monday, October 21 at 12pm Central so that we can send it out via TinyLetter (and post on the blog) Tuesday, October 22. 👻

Thanks!

### CODE REVIEW

File: published/064-2019-oct-22.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Here are some of the highlights since the last IPFS Weekly.
+
+
+## The latest
+
+### IPFS Camp Keynotes and Interviews
+https://blog.ipfs.io/2019-10-14-ipfs-camp-keynotes-interviews/
+
+### WATCH: IPFS Basics Nigh with ProtoSchool Denver
```
Comment (terichadbourne) [inline published/064-2019-oct-22.md]: `Nigh` should be `Night` here (their typo, not yours)

File: published/064-2019-oct-22.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+## Coming up in the Community
+*Did you know IPFS has a community forum at [discuss.ipfs.io](https://discuss.ipfs.io/)? Sign up to participate in discussions about coding, tutorials, see announcements and learn about upcoming community events.*
+
++ **25 October 2019:** [Re­decentra­lize Conference 2019](https://redecentralize.org/events/2019-conference/) Interested in a fairer, freer web and net? Working on decentralisation? Care about privacy, automomy and our future digital infrastructure? Then join us to learn, compare experiences, bounce ideas, share challenges and lessons learnt, hatch collaborations, plan campaigns and start projects. London, UK.
++ **27 October 2019:** [Force Commercial Platform Based on IPFS](https://www.meetup.com/Shanghai-Decentralized-Systems-Meetup-Group/events/265612157/) hosted by IPFS Force, will focus on studying Decentralized Storage Network built on IPFS by Protocol Labs and developing services and application based tools in IPFS bundle. Shanghai, China.
```
Comment (terichadbourne) [inline published/064-2019-oct-22.md]: Suggested rewording to make it more clear that the talk is about IPFS Force, not about IPFS itself: 

`**27 October 2019:** [Force Commercial Platform Based on IPFS](https://www.meetup.com/Shanghai-Decentralized-Systems-Meetup-Group/events/265612157/) This meetup hosted by the IPFS Force Community will introduce the IPFS Force commercial platform, which connects decentralized storage providers and users on top of the IPFS network. Shanghai, China.`

**Review by terichadbourne — CHANGES_REQUESTED** [review]
@renrutnnej Suggested a couple of minor copy changes. I'm not seeing the normal buttons to propose them in a way where you can just accept them directly, sorry!

**Comment by ianopolous** [comment]
@renrutnnej I've added Peergos' latest update and release here:
https://github.com/ipfs/newsletter/pull/127

**Comment by jennwrites** [comment]
This draft is ready for translation @moyid. Thank you!

**Pull request merged**